services/repology/repology-repositories.service.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseSvgScrapingService } = require('..')
+
+const schema = Joi.object({
+  message: Joi.number(),
```

Comment:
We've got some helpers for common schema fields like this one (non-negative number) that we can use here. These reside in [validators](https://github.com/badges/shields/blob/master/services/validators.js)

Just import the validator
```js
const { nonNegativeInteger } = require('../validators')
```
and then change the schema to use it
```suggestion
  message: nonNegativeInteger,
```